modules/bookList.js

```diff
@@ -0,0 +1,55 @@
+const bookList = document.querySelector('.book-list');
+export default class BooksList {
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+  }
+
+  addBook(title, author, id) {
+    if (this.books) {
+      bookList.classList.add('black-border');
+    }
+    this.title = title;
+    this.author = author;
+    this.id = id;
+    this.books.push({ title, author, id });
+    localStorage.setItem('books', JSON.stringify(this.books));
+    this.display(this.title, this.author, this.id);
+  }
+
+  display(title, author, id) {
+    if (this.books) {
+      bookList.classList.add('black-border');
+    }
+    const li = document.createElement('li');
+    li.id = id;
+    li.innerHTML = `<p class = "title-author">"${title}" by ${author} </p>
+      <button class="remove ${id}">Remove</button>`;
+    bookList.appendChild(li);
+    this.remove();
+  }
+
+  remove() {
+    const remove = document.querySelectorAll('.remove');
+    remove.forEach((item) => {
+      item.addEventListener('click', () => {
+        this.books = this.books.filter((book) => {
+          if (book.id !== JSON.parse(item.classList[1])) {
+            return true;
+          }
+
+          return false;
+        });
+        localStorage.setItem('books', JSON.stringify(this.books));
+        if (this.books.length === 0) {
+          bookList.classList.remove('black-border');
+        }
+        const liList = document.querySelectorAll('.book-list li');
+        liList.forEach((item1) => {
+          if (item1.id === item.classList[1]) {
+            item1.remove();
+          }
+        });
+      });
+    });
+  }
```

Comment:
- [ ] Good Job so far your project is awesome and everything works as it should, however, as part of the requirements we should stick with ES6 syntax, kindly update all of your BooksList functions to an arrow functions example : 

```JS
  // Normal Function
  addBook(title, author, id) {
     ...
  }
  // Arrow Function ES6
   addBook = (title, author, id) => {
     ...
  }

```